go/networkqualityd.go

```diff
@@ -2,19 +2,23 @@ package main
 
 import (
 	"bytes"
+	"context"
 	"encoding/json"
 	"flag"
 	"fmt"
 	"html/template"
 	"io"
 	"io/ioutil"
 	"log"
+	"net"
 	"net/http"
+	"os"
+	"os/signal"
 	"strconv"
 	"sync"
 	"time"
 
-	_ "net/http/pprof"
```

Comment:
can you restore this import? Without this, `--debug` is useless.

Reply:
Absolutely! Sorry! Not sure why IDE (`codium`) thought that it could be removed.